( Templates )/Creatures/Shards Red&Gold/world/contentdb/templates/regular/actors/evil/d/shard_add.gas

```diff
@@ -0,0 +1,133 @@
+[t:template,n:shard_gold]
+{
+	category_name = "1W_evil_d";
+	doc = "shard_gold";
+	specializes = base_shard;
+
+	[actor]
+	{
+		[skills]
+		{
+			strength =  20, 0;

```

Comment:
If you took these numbers (str/dex/int/cm, life, mana, xp, damage, defense) from some original monster, it might be worthwhile to comment that here, so that you can later still understand where they come from. Adding these comments usually helps me during balancing ;)